akka-testkit-typed/src/main/scala/akka/typed/testkit/StubbedActorContext.scala

```diff
@@ -181,16 +183,16 @@ class StubbedActorContext[T](
    * Retrieve the inbox representing the given child actor. The passed ActorRef must be one that was returned
    * by one of the spawn methods earlier.
    */
-  def childInbox[U](child: ActorRef[U]): Inbox[U] = {
-    val inbox = _children(child.path.name).asInstanceOf[Inbox[U]]
+  def childInbox[U](child: ActorRef[U]): TestInbox[U] = {
+    val inbox = _children(child.path.name).asInstanceOf[TestInbox[U]]
     if (inbox.ref != child) throw new IllegalArgumentException(s"$child is not a child of $this")
     inbox
   }
 
   /**
    * Retrieve the inbox representing the child actor with the given name.
    */
-  def childInbox[U](name: String): Inbox[U] = _children(name).asInstanceOf[Inbox[U]]
+  def childInbox[U](name: String): TestInbox[U] = _children(name).asInstanceOf[TestInbox[U]]
```

Comment:
is this public?

Reply:
I've made this entire class private[akka] now as i don't think they should know how we do the testkit